snapcraft/commands/account.py

```diff
@@ -88,7 +88,7 @@ def fill_parser(self, parser: "argparse.ArgumentParser") -> None:
             action="store_true",
             default=False,
             help=(
-                "Deprecated option to enable candid login. "
```

Comment:
I needed to shave 5 characters from this help text.  Feel free to provide another suggestion.

Reply:
I prefer this in general :-)